fhirclient/client.py

```diff
@@ -240,3 +243,97 @@ def from_state(self, state):
     def save_state (self):
         self._save_func(self.state)
 
+    # MARK: Pagination
+    def fetch_next_page(self, bundle):
+        """
+        Fetch the next page of results using the `next` link provided in the bundle.
+
+        Args:
+            bundle (Bundle): The FHIR Bundle containing the `next` link.
+
+        Returns:
+            Bundle: The next page of results as a FHIR Bundle.
+        """
+        next_link = self.get_next_link(bundle)
+        if next_link:
+            sanitized_next_link = self.sanitize_next_link(next_link)
+            return self.execute_pagination_request(sanitized_next_link)
+        return None
+
+    def get_next_link(self, bundle):
```

Comment:
Can this method and the other helpers (sanitize_next_link & execute_pagination_request) be marked as private methods with a leading single underscore? They don't strike me as being necessary for the public API and if we don't need them, I'd like to make it clear what's public and what's not.